### PR TITLE
New Logo

### LINKED ISSUE #51: Rough logo sketches suggestion/ideas
### Role

I play modded Minecraft

### Suggestion

2 rough logo designs

### Benefit

No logo yet

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

image

### PR DESCRIPTION
**Pull request by Scrumplex**
Closes #51

### CODE REVIEW

**Comment by catgirldoll** [comment]
Currently has stitching problems. WIP

**Comment by AshtakaOOf** [comment]
icon-monochrome-dark
icon-monochrome-light
Here are the minimal one for various purposes
They should be fixed

**Comment by Sea-Kerman** [comment]
Hmm, the minimal/monochrome ones don’t look very similar to the colored ones, perhaps add dividing gaps/differences in brightness to emulate the dirt block and spectrum

File: launcher/ui/dialogs/AboutDialog.cpp [hunk 1]
```diff
@@ -97,12 +97,19 @@ QString getCreditsHtml()
...
+    stream << QString("<p>Pankakes</p>\n");
+    stream << QString("<p>tobimori %1</p>\n")       .arg(getGitHub("tobimori"));
     stream << "<p>Orochimarufan &lt;<a href='mailto:orochimarufan.x3@gmail.com'>orochimarufan.x3@gmail.com</a>&gt;</p>\n";
     stream << "<p>TakSuyu &lt;<a href='mailto:taksuyu@gmail.com'>taksuyu@gmail.com</a>&gt;</p>\n";
     stream << "<p>Kilobyte &lt;<a href='mailto:stiepen22@gmx.de'>stiepen22@gmx.de</a>&gt;</p>\n";
     stream << "<p>Rootbear75 &lt;<a href='https://twitter.com/rootbear75'>@rootbear75</a>&gt;</p>\n";
     stream << "<p>Zeker Zhayard &lt;<a href='https://twitter.com/zeker_zhayard'>@Zeker_Zhayard</a>&gt;</p>\n";
-    stream << "<p>Everyone else who <a href='https://github.com/PolyMC/PolyMC/graphs/contributors'>contributed</a>!</p>\n";
+    stream << "<p>Everyone who helped establish our branding!</p>\n";
+    stream << "<p>And everyone else who <a href='https://github.com/PolyMC/PolyMC/graphs/contributors'>contributed</a>!</p>\n";
```
Comment (TayouVR) [inline launcher/ui/dialogs/AboutDialog.cpp]:
```suggestion
    stream << "<p>And everyone else who <a href='https://github.com/PrismLauncher/PrismLauncher/graphs/contributors'>contributed</a>!</p>\n";
```
maybe update this already?

**Review by AshtakaOOf — CHANGES_REQUESTED** [review]
Replace the monochrome with the the tiny monochrome?

**Comment by NotNite** [comment]
This line in the README: `Logo and branding also coming soon.` should be removed.

**Comment by Scrumplex** [comment]
> Replace the monochrome with the the tiny monochrome?

Monochrome ones are not used right now. Just putting them into the repo, so we don't lose them. :D

**Review by DioEgizio — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]
Just so that this is ready to merge, i think we still have a few slight logo modifications

**Comment by DioEgizio** [comment]
conflicting with develop

**Comment by jn64** [comment]
The logotype in the source svgs is already rendered to paths. Would be good to document the font used for future reference.

**Comment by catgirldoll** [comment]
The font is custom.

**Pull request merged**